text/0000-spec-api-branches.md

```diff
@@ -0,0 +1,58 @@
+# Meta
+[meta]: #meta
+- Name: Spec API branches
+- Start Date: 2020-01-27
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The version of the specification is represented by two numbers, the Buildpack API and the Platform API. When there are major changes to these APIs the API number is bumped. Rather than requiring a bump to the API for each change that enters the spec, this RFC proposes adding protected branches for future API version to the spec repo. Breaking changes can be PR'd to these branches and the branches themselves will be merged to master when the core team decided that a group of changes together shall constitute the next API version.
+
+# Motivation
+[motivation]: #motivation
+
+Changes to the Platform or Buildpack API can be disruptive, especially right now when both are pre-stable (0.x) and every change is treated as breaking. 
+
+When we want to make several changes to a given API in short succession (i.e. in the time windows between subsequent releases of the reference lifecycle), it would be nice to use a single new API number to represent a set of breaking changes.
+
+Given that we have decided that master of the spec repo shall always describe a specific set of API versions, we should stage changes on a branch, and merge that branch to master when we want to assign a number to the set of changes.
+
+# What it is
+[what-it-is]: #what-it-is
+
+The spec repo shall have protected branches representing the next API version (right now those would be `platform/0.3` and `buildpack/0.3`). PRs to the spec that change an API (all non-cosmetic changes) should be made to those branches.
+
+When they decide it is appropriate, the core team, in consultation with the implementation team will merge an API branch to master, bump the API versions, in the spec README (https://github.com/buildpacks/spec#api-versions), and apply a [tag](https://github.com/buildpacks/spec/releases) to master. 
```

Comment:
Since there are two API versions here, should we clarify/document how we name the tags?

Reply:
updated, have a look